File: android2_lesson01/src/java/java/com/example/ama/android2_lesson01/ui/details/mvp/base/DetailsNoteView.java
package com.example.ama.android2_lesson01.ui.details.mvp.base;

import com.example.ama.android2_lesson01.ui.base.NoteView;
import com.example.ama.android2_lesson01.ui.details.mvp.DetailsNotePresenter;

/**
 * Interface for work with {@link DetailsPresenter}
 */

public interface DetailsNoteView extends NoteView {
    void finishEditing();
}

File: android2_lesson01/src/kotlin/java/com/example/ama/android2_lesson01/ui/details/mvp/base/DetailsNoteView.kt
package com.example.ama.android2_lesson01.ui.details.mvp.base

import com.example.ama.android2_lesson01.ui.base.NoteView

/**
 * Interface for work with [DetailsPresenter]
 */

interface DetailsNoteView : NoteView {
    fun finishEditing()
}